railties/test/application/console_test.rb

```diff
@@ -250,6 +250,32 @@ def foo
     write_prompt "foo", "=> \"this is foo\""
   end
 
+  def test_rails_console_methods_patch_with_module_inclusion_deprecation_caller_location
+    add_to_config <<-RUBY
+      module MyConsole
+        def foo
+          "this is foo"
+        end
+      end
+
+      console do
+        ::Rails::ConsoleMethods.include(MyConsole)
+      end
+    RUBY
+
+    spawn_console("-e development", wait_for_prompt: false)
+
+    line_number = 0
+    app = File.readlines("#{app_path}/config/application.rb")
+    app.each_with_index do |line, index|
+      if line.include?("Rails::ConsoleMethods.include(MyConsole)")
+        line_number = index + 1
+      end
+    end
+
+    assert_output "(called from block in <class:Application> at #{app_path}/config/application.rb:#{line_number})", @primary, 30
```

Comment:
Let's just merge this additional assertion with `test_rails_console_methods_patch_backward_compatibility_with_module_inclusion` to keep fewer of these slow tests around.